spec/API_specification/index.rst

```diff
@@ -3,6 +3,18 @@
 API specification
 =================
 
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
```

Comment:
```suggestion
A conforming implementation of the array API standard must provide and support the APIs and behavior detailed in this specification while adhering to the following conventions.
```

Updating away from "functions", as the placement of this guidance applies to the whole spec, and no longer to a list of functions, as was the case previously.